specification/authorization/resource-manager/readme.md

```diff
@@ -94,6 +94,32 @@ directive:
 ```
 
 
+### Tag: package-2024-09-01-preview
+
+These settings apply only when `--tag=package-2024-09-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-09-01-preview'
```

Comment:
>package-2024-09-01-preview [](http://example.com/codeflow?start=19&length=26)

The first commit needs to be an exact copy of the previous API version. All new changes should only be added in the subsequent commits. 

This allows the reviewer to get a clear understanding of the actual changes being introduced. With the way the PR is raised now, it is not possible for the reviewer to tell what the changes are. 

Please either abandon the PR and raise another one with the recommendation or create a new set of commits on this PR following the recommendation. If you are doing this pattern please indicate which commit is the exact copy of the previous version.

Reply:
Thanks @razvanbadea-msft . I submitted another pull request for same code changes with two commits (1) fully copy the old version and (2) update the version date and schema

PR link https://github.com/Azure/azure-rest-api-specs/pull/30836

Reply:
I have approved the other PR, pl abandon this one